### ISSUE TITLE
Another weirdness with Cinnamon panel icons

### ISSUE THREAD

**Issue by Qronikarz** [linuxmint/mint22.1-beta#97]
Something I noticed in this beta, but can't try in normal now.

1. Some icons have different right click menu styles:
a) date, sound, network - new gray
b) flatpak programs -  old gray
c) bluetooth, update manager - old gray
![sound](https://github.com/user-attachments/assets/0868013d-f879-424e-b3e3-ca9eb65e1c66)
![flatpakheroic](https://github.com/user-attachments/assets/3a264f13-06b2-4f89-b881-ba552abaf733)
![update](https://github.com/user-attachments/assets/843c27d4-1ee1-4836-b09e-f225e4d16b4d)

(the sizes of flatpak icons were reported here - https://github.com/orgs/linuxmint/discussions/411)

2. PrntScrn keyboard button doesn't work after you right click b) and c) from above point, but it works after right clicking a) from above point. I had to launch the screenshot program and set a timer to make those screenshots. I do remember I had the same exact behavior with menu button in Bottles after it got ported to GTK4, but do the panel icons use GTK4?

**Comment by JosephMcc** [comment]
The difference is the source of the menus. Some from applets are provided and themed by Cinnamon. Some are from apps in your system tray. Those are provided by the toolkit the app is built with like Gtk. Those are also themed by the Gtk theme. There isn't a lot that can be done about that if you want to have a system tray.

The screenshot thing is down to how the menus work. Some are requiring a grab of inputs to work properly. Again, that's the nature of the beast.

None of this is new to the beta.

**Comment by Qronikarz** [comment]
Well okay, but will the styles get updated before the beta ends to be visually coherent with the new Cinnamon style?

**Comment by JosephMcc** [comment]
No. They never have matched and can't match exactly because they use different toolkits. The color schemes used by default in Mint haven't changed at all in 22.1

**Comment by Qronikarz** [comment]
> They never have matched

Exactly no, but before they were much closer in the looks.
Here's how it looks in Mint 22.1:
![styling221](https://github.com/user-attachments/assets/1db14dd1-207a-493e-84d6-c0ec4f63ee2a)

Here's how it looks on laptop in Mint 22:
![styling22](https://github.com/user-attachments/assets/f6b8c8ad-fe85-470e-9219-689482e11fe8)
![stylingnemo](https://github.com/user-attachments/assets/ae8dc130-99dc-46cc-ac05-914e229d2ac6)

Panel, menu and nemo looked nearly identical in Mint 22. And I also found this: https://www.reddit.com/r/linuxmint/comments/1hb1z3z/visual_inconsistency_in_linux_mint_221_beta/

> The color schemes used by default in Mint haven't changed at all in 22.1

But the panel, menu and some applets did change their colors.

**Comment by JosephMcc** [comment]
Ah. You are right. I don't use Mint-Y anymore and forgot those colors were changed. I'll let clem answer that one.

**Comment by clefebvre** [comment]
A revamp of the GTK colors is on the roadmap for 22.2.
